packages/postgres/src/index.ts

```diff
@@ -1,12 +1,22 @@
 import type { QueryResult, QueryResultRow } from '@neondatabase/serverless';
 import { type VercelPool, createPool } from './create-pool';
-import type { Primitive } from './sql-template';
+import { tql } from './sql-template';
 
 export * from './create-client';
 export * from './create-pool';
 export * from './types';
 export { postgresConnectionString } from './postgres-connection-string';
 
+export const {
+  fragment,
+  identifiers,
+  list,
+  values,
+  set,
+  unsafe,
+  query: debug,
```

Comment:
maybe this is just me, but i expected `debug` to behave differently. Here is how:
1. I run into a bad query and get some neon db error i don't understand
2. I go into the code and switch `sql` with `debug`
3. get the query logged or thrown

instead the return type of `debug` is different to `sql`, so TS is unhappy and also my runtime error changed because for example i can't read `.rows` on the result of `debug`. Does it make sense to wrap this with a little function to enhance DX event more?

Reply:
Hmm, that might be a good call... log the query and the parameters (with a check on something like `process.env.DEVELOPMENT` or something) and still send the query to the database...